ibis/expr/types.py

```diff
@@ -188,6 +190,55 @@ def op(self):
     def _factory(self):
         return type(self)
 
+    def _find_backends(self):
```

Comment:
next version can you type the outputs here

Reply:
> next PR can you type the outputs here

I think it's worth making mypy run properly before spending much time in annotating new things. I created #2823, I'll see if I can spend some time on it once I finish the bigger part of the refactoring, which is almost over.